nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -47,18 +48,11 @@
 import com.sun.source.tree.Tree;
 import com.sun.source.tree.VariableTree;
 import com.sun.source.util.TreePath;
-import com.sun.source.util.Trees;
 import com.sun.tools.javac.code.Symbol;
-import com.sun.tools.javac.processing.JavacProcessingEnvironment;
 import com.sun.tools.javac.tree.JCTree.JCCompilationUnit;
 import com.sun.tools.javac.util.DiagnosticSource;
 import com.sun.tools.javac.util.JCDiagnostic.DiagnosticPosition;
-import com.uber.nullaway.fixserialization.FixSerializationConfig;
-import com.uber.nullaway.fixserialization.Serializer;
-import com.uber.nullaway.fixserialization.location.AbstractFixLocation;
-import com.uber.nullaway.fixserialization.location.FixLocation;
-import com.uber.nullaway.fixserialization.out.ErrorInfo;
-import com.uber.nullaway.fixserialization.out.SuggestedFixInfo;
+import com.uber.nullaway.fixserialization.SerializationHandler;
```

Comment:
Again, probably a different name, but love how much cleaner this is in terms of changes to core NullAway! :)

Reply:
Renamed to `SerializationService`, please let me know if you still think I should rename it to something else.